recipes/b2/portable/conanfile.py

```diff
@@ -1,8 +1,11 @@
-from conans import ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
 import os
+from contextlib import contextmanager
+import conan.tools.files
+import conan.tools.layout
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.52.0"
```

Comment:
```suggestion
required_conan_version = ">=1.47.0"
```

More than enough: https://github.com/conan-io/conan-center-index/blob/master/docs/v2_linter.md#import-tools-from-conan